cli/src/cli.rs

```diff
@@ -754,25 +754,18 @@ pub fn parse_command(
         ("airdrop", Some(matches)) => {
             let faucet_port = matches
                 .value_of("faucet_port")
-                .unwrap()
+                .ok_or_else(|| CliError::BadParameter("Missing faucet port".to_string()))?
```

Comment:
Maybe we just use `...required(true)`? Or is there any reason for not doing so?

Reply:
There's a default value specified for this parameter on line 2,430 but here on line 757 there's no compile time guarantee that the value of "faucet_port" will not be None hence the error handling - far better for the user to see that error message then have the cli tool panic.

But given the default value, the missing faucet port error should never occur.